lms/envs/common.py

```diff
@@ -1039,6 +1041,48 @@
 ################################# Jasmine ##################################
 JASMINE_TEST_DIRECTORY = PROJECT_ROOT + '/static/coffee'
 
+
+######################### Branded Footer ###################################
+# Constants for the footer used on the site and shared with other sites
+# (such as marketing and the blog) via the branding API.
+
+# URL for OpenEdX displayed in the footer
+FOOTER_OPENEDX_URL = "http://open.edx.org"
+
+# URL for the OpenEdX logo image
+# We use logo images served from files.edx.org so we can (roughly) track
+# how many OpenEdX installations are running.
+# Site operators can choose from these logo options:
+# * https://files.edx.org/openedx-logos/edx-openedx-logo-tag.png
+# * https://files.edx.org/openedx-logos/edx-openedx-logo-tag-light.png"
+# * https://files.edx.org/openedx-logos/edx-openedx-logo-tag-dark.png
+FOOTER_OPENEDX_LOGO_IMAGE = "https://files.edx.org/openedx-logos/edx-openedx-logo-tag.png"
+
+# This is just a placeholder image.
+# Site operators can customize this with their organization's image.
+FOOTER_ORGANIZATION_IMAGE = "images/default-theme/logo.png"
+
+# These are referred to both by the Django asset pipeline
+# AND by the branding footer API, which needs to decide which
+# version of the CSS to serve.
+FOOTER_CSS = {
+    "openedx": {
+        "ltr": "style-lms-footer",
+        "rtl": "style-lms-footer-rtl",
+    },
+    "edx": {
+        "ltr": "style-lms-footer-edx",
+        "rtl": "style-lms-footer-edx-rtl",
+    },
+}
+
+# Cache expiration for the version of the footer served
+# by the branding API.
+FOOTER_CACHE_TIMEOUT = 5 * 60
+
+# Max age cache control header for the footer (controls browser caching).
+FOOTER_BROWSER_CACHE_MAX_AGE = 5 * 60
```

Comment:
@feanil What do you think?  These settings control the server-side and browser caching of the footer accessed through an API end-point.